tests/AbleStoreNonBackedEnumTest.php

```diff
@@ -4,11 +4,14 @@
 
 use Spatie\Activitylog\Test\Enum\NonBackedEnum;
 use Spatie\Activitylog\Test\Models\Activity;
+use Spatie\Activitylog\Test\Models\User;
 
 it('can store non backed enum', function () {
     $description = 'ROLE LOG';
 
-    activity()->withProperty('role', NonBackedEnum::User)->log($description);
+    activity()
+        ->performedOn(User::first())
+        ->withProperty('role', NonBackedEnum::User)->log($description);
```

Comment:
The failing test is because of the enum itself. You will have to skip the test for `PHP < 8.1` 
https://pestphp.com/docs/skipping-tests